src/dev/ci_setup/checkout_sibling_es.sh

```diff
@@ -86,6 +88,7 @@ function checkout_sibling {
 }
 
 checkout_sibling "elasticsearch" "${PARENT_DIR}/elasticsearch" "USE_EXISTING_ES"
+export TEST_ES_FROM=source
```

Comment:
Let's add a note here to indicate this is forcing us to always run from source for the time being.